chains/evm/executor/executor.go

```diff
@@ -149,25 +167,28 @@ func (e *Executor) watchExecution(ctx context.Context, cancelExecution context.C
 				}
 
 				signatureData := sigResult.(*common.SignatureData)
-				hash, err := e.executeProposal(proposals, signatureData)
+				hash, err := e.executeProposal(ctxWithSpan, proposals, signatureData)
 				if err != nil {
-					_ = e.comm.Broadcast(e.host.Peerstore().Peers(), []byte{}, comm.TssFailMsg, sessionID)
+					_ = e.comm.Broadcast(ctxWithSpan, e.host.Peerstore().Peers(), []byte{}, comm.TssFailMsg, sessionID)
+					span.SetStatus(codes.Error, fmt.Errorf("executing proposel has failed %w", err).Error())
 					return err
 				}
 
-				log.Info().Str("SessionID", sessionID).Msgf("Sent proposals execution with hash: %s", hash)
+				logger.Info().Str("SessionID", sessionID).Msgf("Sent proposals execution with hash: %s", hash)
```

Comment:
Shouldn't this also go to trace?

Reply:
I wanted to leave this all for now together with Traces. But now I think maybe let's make all these types of logs DEBUG?

Reply:
made it as debug so we can filtrate these type of logs easier then